app/services/user.js

```diff
@@ -312,20 +315,13 @@ const unsetMedia = async id => {
 const signIn = async username => {
     const notFoundMessage = `The user with username of "${username}"` +
                             ' does not exist.'
-    const badRequestMessage = `The user with username of "${username}"` +
-                              ' is already signed in.'
     const okMessage = `The user with username of "${username}"` +
                       ' was signed into the application.'
     try {
         const user = await User.findOne({ username })
         if (!user) return quickResponse(404, notFoundMessage)
 
-        // check if user is already signed in
-        if (user.active) return quickResponse(400, badRequestMessage)
-
-        user.active = true
```

Comment:
The whole point of this was to implement some sort of "indicator" underneath the user image in the front end. WebSockets may be a better approach for this later on.